BREAKING_CHANGES.md

```diff
@@ -2,6 +2,19 @@
 
 ## v2.0.0
 
+### Remove string output from `dateChange` event in `ix-date-picker`
+
+### Change `onDateChange` event to `DateChangeEvent` in `ix-date-picker`
+
+### Change `getCurrentDate` method return property names in `ix-date-picker`
+
+### Remove deprecated properties from `ix-date-picker`
+
+### 'ix-time-picker' removed deprecated properties
+
+- show-time-reference
+- individual
+
```

Comment:
This should not contain any breaking changes regarding the old components.

Reply:
> This should not contain any breaking changes regarding the old components.
resolved